openapi.yaml

```diff
@@ -16274,6 +16318,69 @@ components:
             The value must be a [Longview Subscription ID](/api/v4/longview-subscriptions) or `null`. Note that a value of `null` will cancel the Longview subscription.
           example: longview-30
           x-linode-cli-display: 1
+    Maintenance:
+      type: object
+      description: >
+        Information about maintenance affecting an entity.
+      properties:
+        type:
+          type: string
+          enum:
+          - reboot
+          - cold_migration
+          - live_migration
+          description: >
+            The type of maintenance.
+          example: reboot
+        status:
+          type: string
+          enum:
+          - pending
+          - ticketed
```

Comment:
The public list of statuses is [pending, ready, started, completed]. The other 'early' statuses are combined into pending and cancelled isn't shown at all.

Reply:
updated